os_migrate/roles/import_workloads/tasks/conv_host.yml

```diff
@@ -22,6 +22,13 @@
         - os_src_conversion_host_info.openstack_conversion_host.status != "ACTIVE"
         - item["_migration_params"]["data_copy"]|default(true)
 
+    - name: fail if src conversion host used when data_copy is false
```

Comment:
Expecting this to fail here if we're still running conversion hosts. I think we are.

Reply:
Migrations are taking a while, recent failure doesn't cover the scenario we're planning for or trying to create here, https://gitlab.cee.redhat.com/os-migrate/bot/-/jobs/27493481/raw

Reply:
Updated code to test for specific scenario